apps/expo-go/ios/Exponent/Versioned/Core/Modules/Notifications/ExpoGoNotificationsSchedulerModule.swift

```diff
@@ -21,8 +21,26 @@ public final class ExpoGoNotificationsSchedulerModule: SchedulerModule {
     contentInput: [String: Any],
     triggerInput: [String: Any]?
   ) throws -> UNNotificationRequest? {
+    var mutableContentInput = contentInput
+    
+    if var data = mutableContentInput["data"] as? [String: Any] {
+      data["experienceId"] = scopeKey
+      data["scopeKey"] = scopeKey
+      mutableContentInput["data"] = data
+    } else {
+      mutableContentInput["data"] = [
+        "experienceId": scopeKey,
+        "scopeKey": scopeKey
+      ]
+    }
+    
+    if let categoryIdentifier = mutableContentInput["categoryIdentifier"] as? String {
```

Comment:
this compensates for the removal of https://github.com/expo/expo/blob/08b2db662bb0b171b649a2e790cd237549322ccd/apps/expo-go/ios/Exponent/Versioned/Core/UniversalModules/EXNotifications/EXScopedNotificationBuilder.m#L35